### PR TITLE
feat(report): CERT notification to slack

### LINKED ISSUE #742: Display CERT information to slack notification
Related #741 #740

### PR DESCRIPTION
**Pull request by kazuminn**
Fixes #742 
# What did you implement:

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.
* When there is a link, it is displayed as JPCERT.
* When there is no link, it is not displayed as USCERT.

# Checklist:
You don't have to satisfy all of the following.

- [ ] Write tests
- [ ] Write documentation
- [x] Check that there aren't other open pull requests for the same issue/feature
- [x] Format your source code by `make fmt`
- [x] Pass the test by `make test`
- [ ] Provide verification config / commands
- [x] Enable "Allow edits from maintainers" for this PR
- [x] Update the messages below

***Is this ready for review?:***YES  

# Reference

* https://blog.github.com/2015-01-21-how-to-write-the-perfect-pull-request/

### CODE REVIEW

File: report/slack.go [hunk 1]
```diff
@@ -327,16 +327,84 @@ func attachmentText(vinfo models.VulnInfo, osFamily string, cweDict map[string]m
...
+				strings.Join(vectors, "\n"),
+				jpalert,
+				"JPCERT",
+				usalert,
+				"USCERT",
+				vinfo.Summaries(config.Conf.Lang, osFamily)[0].Value,
+				mitigation,
+				cweIDs(vinfo, osFamily, cweDict),
+			)
+		} else {
```
Comment (kotakanbe) [inline report/slack.go]: __[Golint]__ if block ends with a return statement, so drop this else and outdent its block

<sub>posted by Sider</sub>

**Comment by kotakanbe** [comment]
@kazuminn Please paste the test result.

**Comment by kazuminn** [comment]
@kotakanbe I tested it with the following pattern in the test.
- When there is a link, it is displayed as JPCERT.
- When there is no link, it is not displayed as USCERT.

File: report/slack.go [hunk 2]
```diff
@@ -327,12 +327,30 @@ func attachmentText(vinfo models.VulnInfo, osFamily string, cweDict map[string]m
...
+		jpalert = ja.URL
+	}
+	var usalert string
+	for _, en := range vinfo.AlertDict.En {
+		usalert = en.URL
+	}
+	if len(jpalert) != 0 && len(usalert) == 0 {
+		certAlert = fmt.Sprintf("<%s|JPCERT>", jpalert)
+	} else if len(usalert) != 0 && len(jpalert) == 0 {
+		certAlert = fmt.Sprintf("<%s|USCERT", usalert)
```
Comment (kotakanbe) [inline report/slack.go]: <%s|USCERT"

You should do the test carefully

File: report/slack.go [hunk 3]
```diff
@@ -327,12 +327,30 @@ func attachmentText(vinfo models.VulnInfo, osFamily string, cweDict map[string]m
 			vinfo.Mitigations(osFamily)[0].Value)
 	}
 
-	return fmt.Sprintf("*%4.1f (%s)* %s %s\n%s\n```\n%s\n```%s\n%s\n",
+	certAlert := ""
+	var jpalert string
+	for _, ja := range vinfo.AlertDict.Ja {
```
Comment (kotakanbe) [inline report/slack.go]: One CVE has multiple alerts.
Your code overwrites the alert, so your code can not show when One CVE has multiple alerts.

Comment (kotakanbe) [reply report/slack.go]: ex. https://github.com/future-architect/vuls/blob/master/alert/cve_jp.go#L76

**Pull request closed without merging**